# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1490103229 14400
#      Tue Mar 21 09:33:49 2017 -0400
# Node ID f77d840f0f7c2d88c23c991c7ddee41c72d79c16
# Parent  d05f688ae836dceddb765e670999849830129d0e
new assignment conversion (bug #50574)

* op-fcs-fs.cc (install_fcs_fs_ops): New assignment conversion for
assignment of octave_scalar to octave_float_complex.

diff --git a/libinterp/operators/op-fcs-fs.cc b/libinterp/operators/op-fcs-fs.cc
--- a/libinterp/operators/op-fcs-fs.cc
+++ b/libinterp/operators/op-fcs-fs.cc
@@ -160,9 +160,12 @@ install_fcs_fs_ops (void)
   INSTALL_CATOP (octave_float_complex, octave_float_scalar, fcs_fs);
   INSTALL_CATOP (octave_complex, octave_float_scalar, cs_fs);
   INSTALL_CATOP (octave_float_complex, octave_scalar, fcs_s);
 
   INSTALL_ASSIGNCONV (octave_float_complex, octave_float_scalar,
                       octave_float_complex_matrix);
   INSTALL_ASSIGNCONV (octave_complex, octave_float_scalar,
                       octave_complex_matrix);
+
+  INSTALL_ASSIGNCONV (octave_float_complex, octave_scalar,
+                      octave_float_complex_matrix);
 }
